### ISSUE TITLE
Unable to load custom files from disk

### ISSUE THREAD

**Issue by efab2022** [xylo/intellij-postfix-templates#220]
Steps to reproduce the problem:

1. import an existing file by selecting "local"
2. the file appears in the editor
3. ![imatge](https://user-images.githubusercontent.com/114433210/194075048-8df085c9-0498-4b5e-b456-686895bae74b.png)
4. but the contents are empty
5. the file does have contents
6. ![imatge](https://user-images.githubusercontent.com/114433210/194075277-7503f695-07d6-4197-b9de-826f9b75c5b1.png)

Expected behaviour: the editor should show the contents of the file, and the templates should be loaded and work
Actual behaviour: the editor is empty, and none of the defined templates work.

Right now the only workaround is to "Edit the customization of current language". But this is super obscure and the more obvious way should also work, out of the box.

**Comment by kinceeR** [comment]
I also have some problem like yours

**Comment by kinceeR** [comment]
but it already solved after comfirm “apply” btn,then is shown

**Comment by xylo** [comment]
I agree, the need for pressing the apply button first ist noch very intuitive.

**Comment by efab2022** [comment]
Thanks for the reply. Can you guide me through the steps to fix this? I tried clicking "apply" first as well and it doesn't work.. even after removing and re-adding the file, and clicking on apply first, no content is loaded. Is there a step-by-step documented way to load file from the local file system? tx :)

**Comment by Taoaozw** [comment]
> Thanks for the reply. Can you guide me through the steps to fix this? I tried clicking "apply" first as well and it doesn't work.. even after removing and re-adding the file, and clicking on apply first, no content is loaded. Is there a step-by-step documented way to load file from the local file system? tx :)

i also cant load ,my env cant connect to the public network,Whether it makes a difference

**Comment by anneborcherding** [comment]
For me, the workaround with the "Apply" button did not resolve the issue, but when I included the file as "User template file" instead of "Local file" it was loaded correctly (after clicking on a different template and then clicking back on my new template again)
